ftw/publisher/sender/tests/assets/folder_dx.json

```diff
@@ -1,5 +1,6 @@
 {
     "unicode:backreferences_adapter": {},
+    "unicode:constrain_types_adapter": {"utf8:mode": -1},
```

Comment:
Same as in receiver I'm not quite sure what the constrain does here.

Reply:
It's a consequence of https://github.com/4teamwork/ftw.publisher.core/pull/66
this file is the expected results of test_dexterity_folder_extractor